### PR TITLE
data: ismailwafid

### PR DESCRIPTION
**Pull request by ismailwafid**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8727"></a>

### CODE REVIEW

**Review by accodes21 — CHANGES_REQUESTED** [review]
Please ```.json``` to your filename.

File: ismailwafid.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/WafidIsmail",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/ismailwafid",
+      "icon": "FaGithub"
+    },
```
Comment (sital002) [inline ismailwafid.json:14]:
```suggestion
    }
```

**Review by sital002 — CHANGES_REQUESTED** [review]
The trailing comma isnot required. Please remove it.

**Review by sital002 — APPROVED** [review]
Looks good 👍
Welcome to Eddie Hub Community 🎉🎉

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Your file should be in `data` directory. Please rename your [file here](https://github.com/ismailwafid/LinkFree/edit/ismailwafid-patch-1/ismailwafid.json) to `data/ismailwafid.json`

Change `ismailwafid.json`  -->  `data/ismailwafid.json`

Let us know if you need any help with this! :)

**Comment by kumarsonsoff3** [comment]
> Can't believe I missed that

It's okay, sital. It does happen! 😄 
Love seeing your amazing PR reviews though! 👍

**Pull request merged**